data/mods/DinoMod/monsters/dinosaur.json

```diff
@@ -868,7 +868,7 @@
     "symbol": "D",
     "color": "magenta_green",
     "volume": "400000 ml",
```

Comment:
Once this is merged I'll create a separate PR to replace "400000 ml" with "volume": "400 L"